_data/publications/1675782.yml

```diff
@@ -2,6 +2,6 @@ inspire-id: 1675782
 project: 
 - madminer
 - exploratory-ml
-link: https://doi.org/10.1073/pnas.1915980117
+link: "https://doi.org/10.1073/pnas.1915980117"
```

Comment:
Quotes are optional. Be design, URLs are always valid implicit strings in YAML. The only strings that must have quotes are ones with `: ` (colon followed by space), and other valid yaml identifiers (a single number, `true`, etc.)

Reply:
thanks, I was a little confused when the : needed a "". I also have `DOI:10.10/...`. so put quotes on those as well. It's the space that I was missing.

Reply:
I merged this myself b/c Peter says it's urgent

Reply:
That's fine! This is a proceedings & slides day for me...